controllers/perconaserverformysql_controller.go

```diff
@@ -243,48 +270,57 @@ func (r *PerconaServerForMySQLReconciler) reconcileReplication(
 
 func reconcileReplicationPrimaryPod(
 	ctx context.Context,
-	cl k8s.APIListPatcher,
+	cl client.Client,
 	cr *apiv2.PerconaServerForMySQL,
 ) error {
+	l := log.FromContext(ctx).WithName("reconcileReplicationPrimaryPod")
+
 	pods, err := k8s.PodsByLabels(ctx, cl, mysql.MatchLabels(cr))
 	if err != nil {
 		return errors.Wrap(err, "get MySQL pod list")
 	}
+	l.Info(fmt.Sprintf("got %v pods", len(pods)))
```

Comment:
It seems like a debug log

Reply:
I make these as debug level. it will be still output but can be easily filtered out